application/templates/pdb.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.pdb.enabled }}
+{{- if or .Values.pdb.minAvailable .Values.pdb.maxUnavailable }}
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: '{{ template "application.name" . }}-pdb'
```

Comment:
Add labels

Reply:
I am not sure about labels section if it allows any.
https://kubernetes.io/docs/tasks/run-application/configure-pdb/

Reply:
these are not label selection. these are labels we add it to every resource created from this helm chart. just to identify that this is part of helm chart

Reply:
hmm..what labels do we usually add? .stakater and .chart?

Reply:
added some, let me know if more labels needed